src/Processors/Executors/ExecutionThreadContext.cpp

```diff
@@ -46,6 +46,9 @@ static void executeJob(ExecutingGraph::Node * node, ReadProgressCallback * read_
 {
     try
     {
+        if (node->processor->spillable() && CurrentThread::getGroup())
```

Comment:
`executeJob()` seems to be a pretty hot function. We'd like to do as few virtual calls from it as possible. Is it possible to replace a virtual call with a simple check of a flag?